frontend/src/Components/Page/Header/SeriesSearchInput.js

```diff
@@ -82,6 +72,14 @@ class SeriesSearchInput extends Component {
       );
     }
 
+    if (item.type === LOADING_TYPE) {
+      return (
+        <LoadingIndicator
+          rippleClassName={styles.ripple}
```

Comment:
This loader looks a little big and out of place and vertically it's not centered (perhaps leading to the out of place part of it).

![image](https://user-images.githubusercontent.com/413544/71774082-2c2e2480-2f1d-11ea-9867-e8d2c0cbe2d2.png)

Reply:
OK I've shrunk the loader from 50 to 30 pixels and added some padding to the container so that the loader takes up the same space as a single series result.  That way if you get an exact match the `Add New Series` title doesn't jump around.